core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionName.java

```diff
@@ -221,7 +221,9 @@ public enum BuiltinFunctionName {
   QUERY(FunctionName.of("query")),
   MATCH_QUERY(FunctionName.of("match_query")),
   MATCHQUERY(FunctionName.of("matchquery")),
-  MULTI_MATCH(FunctionName.of("multi_match"));
+  MULTI_MATCH(FunctionName.of("multi_match")),
+  WILDCARDQUERY(FunctionName.of("wildcardquery")),
```

Comment:
why two wildcard query function defined?

Reply:
This is to support both as they are in supported in the legacy engine.